### PR TITLE
Update to Jenkins 2.121

### PR DESCRIPTION
**Pull request by greg-hellings**
Update to Jenkins LTS 2.121.3
Update plugins to match
Add CHANGELOG entry missing from 1.2.0

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,4 +1,17 @@
-v 1.1.0 (Next)
+v 1.3.0 (20 Sep 2018)
+- Bump default Jeknins to 2.121
```
Comment (tehsmyers) [inline CHANGELOG]: Typo, should be "Jankies", not "Jeknins"

**Review by tehsmyers — DISMISSED** [review]
LGTM, though I didn't care to double-check the plugin list updates (I assume they're just copied in anyway?)

**Comment by greg-hellings** [comment]
They are copied, yes.

File: CHANGELOG (hunk 1 above)
Comment (greg-hellings) [reply CHANGELOG]: Yes, changed to "Jankies".

**Review by abraverm — APPROVED** [review]

**Review by tehsmyers — APPROVED** [review]

**Pull request merged**